xfce-extra/xfce4-verve-plugin/xfce4-verve-plugin-2.0.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A comfortable command line plugin for the Xfce panel"
+HOMEPAGE="https://goodies.xfce.org/projects/panel-plugins/xfce4-verve-plugin"
+SRC_URI="https://archive.xfce.org/src/panel-plugins/${PN}/${PV%.*}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
Check if `GPL-2+` isn't more correct.

Reply:
If you're touching it for the first time, and especially if it's old, GPLs are worth rechecking because people have that wrong often.